ui/src/reusable_ui/components/dropdowns/DropdownItem.tsx

```diff
@@ -0,0 +1,75 @@
+// Libraries
+import React, {Component} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {DropdownChild} from 'src/reusable_ui/types'
+
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+interface Props {
+  itemKey: string
```

Comment:
I think you might be able to just use the `key` prop that's also being passed in. It's confusing to me to have both—is it necessary?